python/cuml/preprocessing/encoders.py

```diff
@@ -174,6 +174,18 @@ def _check_input(self, X, is_categories=False):
             self._set_input_type('df')
             return X
 
+    def _check_input_fit(self, X, is_categories=False):
+        """Helper function used in fit. Can be overridden in subclasses"""
+        return self._check_input(X, is_categories=is_categories)
+
+    def _unique(self, inp):
+        """Helper function used in fit. Can be overridden in subclasses"""
```

Comment:
```suggestion
        """Helper function used in fit. Can be overridden in subclasses. """
        
        # Default implementation passes input through directly since this is performed in `LabelEncoder.fit()`
```

Reply:
Sounds good, let's add this in a separate PR. Thanks!

Reply:
Opened PR #2271.